### DISCUSSION TITLE
Gotta be a way

### DISCUSSION THREAD

**Discussion by thecoolmiker** [3kh0/ext-remover#769] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I was trying to downgrade my chromebook for like an hour and it kept saying out of date, so I tried to turn on dev mode for the recovery screen and see if that would do somthing and not sure what i did after but i turned it off and on or some shit and some how, MY chromebook BOOTED INTO THE LOGIN AS DEV MODE, which is impossible since my school re enrolls and forces you to disable dev mode by popping up the return to secure mode, but it didnt. And like i said not sure what i did. But it went to completely personal chromebook type mode and developer, was worried and re-enrolled myself with my school email, i regret it, but my point is theres gotta be a exploit for enabling developer mode through bugging the recovery mode or something. Throwing this out there if someone wants to tryin figure out whats going on here and make it a exploit. 🤷‍♂️

**Comment by 3kh0** [comment, marked as answer]
Yeah Google has "fixed" downgrading by making sure all recovery images are up to date, this kinda sucks

Sources:
- https://support.google.com/chromebook/thread/220495064?hl=en
- https://www.reddit.com/r/chromeos/comments/5jreu0/chromebook_recovery_media_can_become_outofdate/

**Reply by thecoolmiker** [reply]
Figured so, but still just confused as fuck on how my chromebook went into developer mode, and completely unenrolled 😬
